packages/system-tests/src/spectron/application.ts

```diff
@@ -120,15 +120,28 @@ export class SpectronApplication {
   private async checkWindowReady(): Promise<any> {
     await this.webclient.waitUntilWindowLoaded(60000);
 
-    // Pick the first workbench window here
+    // Pick the window that loads VSCode,
+    // added code to address Spectron limitation in Windows OS (https://github.com/electron/spectron/issues/60)
     const count = await this.webclient.getWindowCount();
-
-    for (let i = 0; i < count; i++) {
-      await this.webclient.windowByIndex(i);
-
-      if (/bootstrap\/index\.html/.test(await this.webclient.getUrl())) {
```

Comment:
This could be a basic question, but why is the window that we want in focus during the Spectron test going to be named "...bootstrap/index.html"? Is that a Spectron convention?